jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -511,6 +511,7 @@ properties:
           lamp_mode:
             title: Lamp operating mode
             type: string
+            enum: [BRIGHTOBJ, FIXEDSLIT, GRATING-ONLY, IFU, MSASPEC, NULL]
```

Comment:
What is the `GRATING-ONLY` mode?

Reply:
From jp-2927:
Comment by James Muzerolle: GRATING_ONLY is a limited special case that is used to move the grating wheel without taking any exposures (no lamp is ever turned on).  You can ignore it for the purposes of processing lamp data.